src/main/java/core/basesyntax/service/impl/BalanceOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import static javax.swing.UIManager.put;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        if (transaction.getQuantity() == 0) {

```

Comment:
what if the quantity is less than zero? Please validate quantity properly in all handlers